MobalMobal/MobalMobal/Main/MainViewController.swift

```diff
@@ -0,0 +1,198 @@
+//
+//  MainViewController.swift
+//  MobalMobal
+//
+//  Created by 김재희 on 2021/02/27.
+//
+
+import SnapKit
+import Then
+import UIKit
+
+class MainViewController: UIViewController {
+    // MARK: - UIComponent
+    let titleView: UIView = {
+        let view: UIView = UIView(frame: .zero)
+        view.backgroundColor = .backgroundColor
+        return view
+    }()
+    
+    let titleLabel: UILabel = {
+        let label: UILabel = UILabel(frame: .zero)
+        label.text = "Hi, jaehui"
+        label.font = UIFont(name: "Futura-Bold", size: 30)
+        label.textColor = .white
+        return label
+    }()
+    
+    let notiListButton: UIButton = {
+        let button: UIButton = UIButton(frame: .zero)
+        button.setImage(UIImage(named: "icMyProfile"), for: .normal)
```

Comment:
```suggestion
        button.setImage(UIImage(named: "icAlarm"), for: .normal)
```
`notiListButton`과 `profileImageView`의 이미지가 서로 바뀐 것 같아요